test/release.spec.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-env jest */
+'use strict'
+const { getOctokit } = require('@actions/github');
+const { getLatestRelease, getUnreleasedCommits } = require('../src/release');
+const { createIssue } = require('../src/utils');
+const { allCommitsData: mock_allCommits,
+  allReleasesData: mock_allReleases,
+  unreleasedCommitsData_0: mock_unreleasedCommitsData_0,
+  unreleasedCommitsData_1: mock_unreleasedCommitsData_1,
+  allReleasesData_NoData: mock_allReleasesData_NoData,
+} = require('./testData')
+
+const token = 'dummytoken'
+
+jest.mock('@actions/github', () => ({
+  getOctokit: jest.fn(),
+  context: { repo: { owner: "sameer", repo: "testrepo" } }
+}))
+
+test("Gets the latest release of the repository", async () => {
+  getOctokit.mockImplementation(() => {
+    return { request: async () => mock_allReleases }
+  });
+  const response = await getLatestRelease(token);
```

Comment:
naming: response is misleading. this function returns the latest release, so I'd call it latestRelease